python/lsst/verify/tasks/commonMetrics.py

```diff
@@ -24,7 +24,12 @@
 """Code for measuring metrics that apply to any Task.
 """
 
-__all__ = ["TimingMetricConfig", "TimingMetricTask"]
+__all__ = ["TimingMetricConfig", "TimingMetricTask",
+           "MemoryMetricConfig", "MemoryMetricTask",
+           ]
+
+import resource
```

Comment:
`psutil` wasn't helpful in the end then?

Reply:
It was helpful for diagnosing what was going on, but reimplementing `@timeMethod` using `psutil` is a bit out of scope for this ticket.